Sparkle/SUBasicUpdateDriver.m

```diff
@@ -3,7 +3,7 @@
 //  Sparkle
 //
 //  Created by Andy Matuschak on 4/23/08.
-//  Copyright 2008 Andy Matuschak. All rights reserved.
```

Comment:
Please don't remove copyright attribution. That's not legal.

Reply:
Thanks, I'm used to updating copyrights in internal-only code. I'll revert this change and leave it untouched for now.